crates/openvino/tests/setup.rs

```diff
@@ -23,6 +25,13 @@ fn read_network() {
 
 #[test]
 fn read_network_from_buffers() {
+    // OpenVINO 2024.2 changed the order of the `ov_element_type_e` enum, breaking compatibility
+    // with older versions. Since we are using 2024.2+ bindings here, we skip this test when
+    // using older libraries.
+    if is_version_pre_2024_2() {
+        return;
```

Comment:
Nit - Maybe print that the test is being skipped and not actually passing so that it still passes for CI.